packages/editor-ui/src/composables/usePushConnection.ts

```diff
@@ -437,6 +437,7 @@ export function usePushConnection({ router }: { router: ReturnType<typeof useRou
 							message: runDataExecutedErrorMessage,
 							type: 'error',
 							duration: 0,
+							dangerouslyUseHTMLString: true,
```

Comment:
Can we just use the same approach as [here](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/Projects/ProjectMoveResourceConfirmModal.vue#L63)

Reply:
Yes let's not use the `dangerouslyUseHTMLString` anymore